src/Nethermind/Nethermind.Blockchain/NullTransactionPool.cs

```diff
@@ -17,47 +17,55 @@
  */
 
 using System;
+using Nethermind.Blockchain.TransactionPools;
 using Nethermind.Core;
 using Nethermind.Core.Crypto;
+using Nethermind.Core.Model;
 using Nethermind.Dirichlet.Numerics;
 
 namespace Nethermind.Blockchain
 {
-    public class NullTransactionStore : ITransactionStore
+    public class NullTransactionPool : ITransactionPool
     {
-        private NullTransactionStore()
+        private NullTransactionPool()
         {
         }
 
-        public static NullTransactionStore Instance { get; } = new NullTransactionStore();
+        public static NullTransactionPool Instance { get; } = new NullTransactionPool();
 
-        public void StoreProcessedTransaction(Keccak txHash, TransactionReceipt receipt)
+        public Transaction[] PendingTransactions => new Transaction[0];
+
+        public TransactionReceipt GetReceipt(Keccak txHash) => null;
+
+        public void AddFilter<T>(T filter) where T : ITransactionFilter
         {
         }
 
-        public TransactionReceipt GetReceipt(Keccak txHash)
+        public void DeleteFilter(Type type)
         {
-            throw new NotSupportedException();
         }
 
-        public AddTransactionResult AddPending(Transaction transaction, UInt256 blockNumber)
+
+        public void AddPeer(ISynchronizationPeer peer)
         {
-            return AddTransactionResult.Added;
         }
 
-        public void RemovePending(Transaction transaction)
+        public void DeletePeer(NodeId nodeId)
         {
         }
 
-        public Transaction[] GetAllPending()
+        public void AddTransaction(Transaction transaction, UInt256 blockNumber)
         {
-            return Array.Empty<Transaction>();
         }
 
-        public event EventHandler<TransactionEventArgs> NewPending
+        public void DeleteTransaction(Keccak hash)
```

Comment:
I would prefere Remove instead of Delete - delete has rather a meaning of deleting from persisted storage and Remove - removing from a collection. It is not I will defend fiercely though.